jfa/types.py

```diff
@@ -3,6 +3,15 @@
 from typing import Optional, TypedDict
 
 
+class ParentFields(TypedDict):
```

Comment:
@jangelsb I know this is nitpicky, but for clarity, my preference would be `ParentIssue*`, not `Parent*`, if you could please make that tweak.

```suggestion
class ParentIssueFields(TypedDict):
```

Reply:
Okay, updated in 35a0843b39d7ec9c0c97d63471ed09efbd81f662.